tests/fuzz/Makefile.fuzz

```diff
@@ -0,0 +1,48 @@
+# Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
From IRC: this file doesn't need to exist anymore now that the Makefile.am works